### PR TITLE
Week 1 and Week 2 updates

### PR DESCRIPTION
**Pull request by gupta-arpan**
## Project

Name of the project

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

### CODE REVIEW

File: docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-07.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- [x] Understanding the QuML editor and architecture 
+- [x] Getting familier with the codebase
+- [x] Understanding the QuML specification and Question Schema
+
+## Screenshots / Videos 
+
+## Contributions
+
+## Learnings
+How to navigate a codebase and work on it.
```
Comment (shubhigupta991) [inline docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-07.md:16]: Please elaborate.

File: docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-21.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Screenshots / Videos 
+<img width="755" alt="17-06-22" src="https://github.com/Code4GovTech/c4gt-milestones/assets/98040726/172d3635-72ad-4ca6-91f7-a00d793ef319"/>
+
+## Contributions
+https://github.com/Sunbird-inQuiry/editor/pull/82/commits/f93a37331c43f2b4589234b1a428cd0fe686c3b4
+https://github.com/Sunbird-inQuiry/editor/pull/82/commits/fd99b99bf434ce8253cea0c4bd406d4714810b54
+https://github.com/Sunbird-inQuiry/editor/pull/82/commits/c0e88fd401f4e1b0ce118435175ed78bec4d3d10
+
+
+## Learnings
```
Comment (shubhigupta991) [inline docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-21.md:20]: Add learnings

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]

File: docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-07.md (hunk 1 above)
Comment (gupta-arpan) [reply docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-07.md]: Done, kindly review it

File: docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-21.md (hunk 2 above)
Comment (gupta-arpan) [reply docs/2023/Sunbird inQuiry/Editor to support new type of questions/updates/2023-07-21.md]: Done, kindly review it.

**Review by shubhigupta991 — APPROVED** [review]

**Pull request merged**